templates/forms/default/data.html.twig

```diff
@@ -22,7 +22,7 @@
                                         {% set use_keys = field.use is defined and field.use == 'keys' %}
                                         {% for key,value in form.value(scope ~ field.name) %}
                                             {% set index = (use_keys ? key : value) %}
-                                            <li>{{ field.options[index]|e }}</li>
+                                            <li>{{ field.options[index]|t|e }}</li>
```

Comment:
This commit doesn't below here, I think you had another issue with this change?

Reply:
I've included it here by intention, as it is related to checkboxes. I think it wont hurt to merge it here, but I can also move it into a separate PR if that is critical.

Reply:
i merged that PR already.

Reply:
The mentioned PR https://github.com/getgrav/grav-plugin-form/pull/475 was about the `select` field. This is about `checkboxes`.